THIRD-PARTY-NOTICES.txt

```diff
@@ -0,0 +1,32 @@
+This software depends on external packages and source code.
+The applicable license information is listed below:
+
+----
+
+File: ResizableArray: revision 8cd063a1e163a0db906343bb5ba1b53c02eb83d6

```

Comment:
Change it to this:

```
-------------------------------------------------------------------------------
| generic name:  CoreFXLab version: 8cd063a1e163a0db906343bb5ba1b53c02eb83d6
| Package URL: pkg:generic/CoreFXLab@8cd063a1e163a0db906343bb5ba1b53c02eb83d6?download_url=https://github.com/dotnet/corefxlab/blob/8cd063a1e163a0db906343bb5ba1b53c02eb83d6/src/System.Collections.Sequences/System/Collections/Sequences/ResizableArray.cs
-------------------------------------------------------------------------------
Homepage: https://github.com/dotnet/corefxlab
Download: https://github.com/dotnet/corefxlab/blob/8cd063a1e163a0db906343bb5ba1b53c02eb83d6/src/System.Collections.Sequences/System/Collections/Sequences/ResizableArray.cs
--- --- --- --- --- --- --- Copyright notice(s) --- --- --- --- --- --- --- ---

Copyright (c) .NET Foundation and Contributors

--- --- --- --- --- --- --- License text(s) --- --- --- --- --- --- --- --- ---

This package is licensed under MIT

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
name: MIT
spdx-key: MIT
https://scancode-licensedb.aboutcode.org/mit
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

The MIT License (MIT)

Copyright (c) .NET Foundation and Contributors

All rights reserved.

Permission is hereby granted, free of charge, to any person obtaining a copy
of this software and associated documentation files (the "Software"), to deal
in the Software without restriction, including without limitation the rights
to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
copies of the Software, and to permit persons to whom the Software is
furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.

THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
SOFTWARE.
```

Reply:
Done